doc/dev/design/FABRID.rst

```diff
@@ -0,0 +1,431 @@
+********
+FABRID
+********
+.. _fabrid-design:
+
+- Author: Justin Rohrer, Jelte van Bommel, Marc Odermatt, Marc Wyss, Cyrill Krähenbühl, Juan A. García-Pardo
+- Last Updated: 2024-04-01
+- Discussion at:
+
+Abstract
+===========
+
+In SCION the endhosts have the option to choose *inter-AS* paths to forward packets to a destination.
+However, some applications require more fine grained path selection like "Do not route traffic over devices
+produced by hardware manufacturer X or that run software Y with version < Z" which requires transparency and
+control also with respect to *intra-AS* paths.
+This is useful for example if there exists a known bug in certain router versions that affect secure communication,
+or if an entity simply does not trust a certain hardware manufacturer.
+This can also be seen as an enhancement for keeping traffic within a certain jurisdiction, e.g., by routing traffic
+only along devices located in a specific country.
+
+Background
+===========
+
+`FABRID <https://netsec.ethz.ch/publications/papers/2023_usenix_fabrid.pdf>`_, is suggested as a solution to the
+aforementioned problem.
+A deployment of FABRID in SCIONLab makes its next-generation features available to interested parties around the globe.
+FABRID also implicitly implements the `EPIC <https://netsec.ethz.ch/publications/papers/Legner_Usenix2020_EPIC.pdf>`_,
+features source authentication for routers and destination host and path validation for source and destination hosts.
+
+Proposal
+========
+
+FABRID indroduces policies, which can be thought of as additional path constraints that should be applied to intra-AS paths.
+The border routers use those policies to decide on the intra-AS path to forward, e.g. by using MPLS labels.
+Some FABRID policies are globally defined and others locally per AS.
+The AS network operator configures which global FABRID policies are supported for the local AS and can add additional local FABRID
+policies that are valid for this AS.
+A source endhost can then select a path together with FABRID policies and forward the FABRID packet over this path to a destination endhost.
+The destination endhost will then be able to recompute the path validator to verify that the packet had been forwarded over that inter-AS path.
```

Comment:
Path validator has not been introduced so far?
Also, it is maybe too detailed for this section. Just say that the routers add some proof of transit such that the source can later verify that the data indeed followed the intended path?